test/spec/adapters/gumgum_spec.js

```diff
@@ -0,0 +1,202 @@
+import {expect} from 'chai';
+import Adapter from '../../../src/adapters/gumgum';
+import bidManager from '../../../src/bidmanager';
+import adLoader from '../../../src/adloader';
+import { STATUS } from '../../../src/constants';
+
+describe('gumgum adapter', () => {
+  'use strict';
+
+  let adapter;
+  let sandbox;
+
+  const TEST = {
+    PUBLISHER_IDENTITY: 'ggumtest',
+    BIDDER_CODE: 'gumgum',
+    PLACEMENT: 'placementId',
+    CPM: 2
+  };
+  const bidderRequest = {
+    bidderCode: TEST.BIDDER_CODE,
+    bids: [{ // in-screen
+      bidId: 'InScreenBidId',
+      bidder: TEST.BIDDER_CODE,
+      placementCode: TEST.PLACEMENT,
+      sizes: [ [728, 90] ],
+      params: {
+        inScreen: TEST.PUBLISHER_IDENTITY
+      }
+    }, { // in-image
+      bidId: 'InImageBidId',
+      bidder: TEST.BIDDER_CODE,
+      placementCode: TEST.PLACEMENT,
+      sizes: [ [728, 90] ],
+      params: {
+        inImage: TEST.PUBLISHER_IDENTITY
+      }
+    }, { // native
+      bidId: 'NativeBidId',
+      bidder: TEST.BIDDER_CODE,
+      placementCode: TEST.PLACEMENT,
+      sizes: [ [728, 90] ],
+      params: {
+        native: 10
+      }
+    }, { // slot
+      bidId: 'InSlotBidId',
+      bidder: TEST.BIDDER_CODE,
+      placementCode: TEST.PLACEMENT,
+      sizes: [ [728, 90] ],
+      params: {
+        inSlot: 10
+      }
+    }, { // no identity
+      bidId: 'NoIdentityBidId',
+      bidder: TEST.BIDDER_CODE,
+      placementCode: TEST.PLACEMENT,
+      sizes: [ [728, 90] ]
+    }]
+  };
+  const bidderResponse = {
+    "ad": {
+      "id": 1,
+      "width": 728,
+      "height": 90,
+      "markup": "<div>some fancy ad</div>",
+      "ii": true,
+      "du": "http://example.com/",
+      "price": TEST.CPM,
+      "impurl": "http://example.com/"
+    }
+  };
+
+  function mockBidResponse(response) {
+    sandbox.stub(bidManager, 'addBidResponse');
+    sandbox.stub(adLoader, 'loadScript');
+    adapter.callBids(bidderRequest);
+    pbjs.handleGumGumCB['InScreenBidId'](response);
+    return bidManager.addBidResponse.firstCall.args[1];
+  }
+
+  beforeEach(() => {
+    pbjs._bidsRequested = [];
```

Comment:
I don't think we need it. I was just trying to follow the convention used on other tests.

I'll remove it shortly.